CODE_OF_CONDUCT.rst

```diff
@@ -0,0 +1,30 @@
+Code of Conduct
+===============
+
+Like the technical community as a whole, the Python-based SVG editor team and community is made up of a mixture of professionals and volunteers from all over the world.
+
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to when they're participating within this community and project. These rules apply equally to founders, mentors and those seeking help and guidance.
+
+This isn't an exhaustive list of things that you can't do. Rather, take it in the spirit in which it's intended - a guide to make it easier to enrich all of us, the technical community and the conferences and usergroups we hope to guide new speakers to.
+
+This code of conduct applies to all communication: the issue tracker, comments, commits, and any other forums created by the project team which the community uses for communication.
+
+If you believe someone is violating the code of conduct, we ask that you report it by emailing bartosz.telenczuk@gmail.com.
```

Comment:
maybe we should ask something about the way people get elected. How about:

1) there could be more than one person to contact, but not more than lets say 5
2) new people can be added to the list with a support of at least two contributors that created at least 2 pull requests in the period of last 2 years. This can be done by adding a pull request and any contributors supporting can comment on the issue.

what do you think?

what do you think?

Reply:
Makes sense. Perhaps we could add a paragraph like this:

If you believe someone is violating the code of conduct, we ask that you report it by contacting any of our community leaders.
     - I imagine, that emails and/or GitHub accounts of community leaders should be listed here.
New members may join as community leaders with the support of at least two contributors, that created at least 2 pull requests in the period of last 2 years. This can be done by adding a pull request and any contributors supporting can comment on the issue.
Community leaders will review your complaint and decide on course of action following the incident.

Reply:
yes, that sounds great to me!

Reply:
Great, then I will be able to update this and send new commit later tonight.